docs/docsource/operation/Introduction/supported_clouds+services.rst

```diff
@@ -0,0 +1,76 @@
+Supported Clouds and Services
+=============================
+
+Zenko replicates and manages data from one or many sources to one or several
+destinations. Sources can be clouds or services, and targets can be of several
+types of cloud backend.
+
+Supported Sources
+-----------------
+
+Zenko supports multiple-cloud storage operations natively using the Amazon
+Simple Storage Service (S3) protocol. It can import data from many S3-compatible
+sources, as well as from servers using supported protocols.
+
+The following Amazon S3-based clouds have been tested as Zenko-compatible
+sources:
+
+*  Amazon Simple Storage Service
+*  Scality RING with S3 Connector
+*  Wasabi Hot Cloud Storage
+*  Ceph RADOS Gateway
+
+In addition to its compatibility with S3 cloud frontends, Zenko can ingest data and
+metadata from servers using the following protocols:
+
+* Scality RING with sproxyd
```

Comment:
This should be S3C not sproxyd.

Reply:
deleted (mentioned in the previous bullet list)